cpp/arcticdb/CMakeLists.txt

```diff
@@ -40,7 +40,8 @@ find_package(zstd CONFIG REQUIRED)  # "CONFIG" bypasses our cpp/CMake/FindZstd.c
 
 find_package(azure-identity-cpp CONFIG REQUIRED)
 find_package(azure-storage-blobs-cpp CONFIG REQUIRED)
-
+find_package(mimalloc REQUIRED)
+message(STATUS "mimalloc version: ${mimalloc_VERSION}")
```

Comment:
Are there any restrictions on the version used? You can add them to the `find_package` call if so

Reply:
I don't think there is any restrictions on the version.